src/pe_reports/charts.py

```diff
@@ -0,0 +1,162 @@
+"""Class methods for report charts."""
+
+# Third-Party Libraries
+import matplotlib.pyplot as plt
+from matplotlib.ticker import MaxNLocator
+
+
+class Charts:
+    """Build charts."""
+
+    def __init__(self, df, width, height, name, title, x_label, y_label):
+        """Initialize chart class."""
+        self.df = df
+        self.title = title
+        self.x_label = x_label
+        self.y_label = y_label
+        self.width = width
+        self.height = height
+        self.name = name
+
+    def pie(self):
+        """Build pie chart."""
+        df = self.df
+        width = self.width
+        height = self.height
+        name = self.name
+        plt.rcParams.update({"figure.max_open_warning": 0})
+        catName = df.columns[0]
+        Val_1Name = df.columns[1]
+        df = df.sort_values(by=Val_1Name, ascending=False)
+        Category_column = df[catName]
+        Value_column = df[df.columns[1]]
+        labels = Category_column
```

Comment:
You have a mix of naming styles with your variables (e.g. `catName` vs. `Category_column`).  You should choose one style and use it consistently.  We don't have it codified in our [style guide](https://github.com/cisagov/development-guide/blob/develop/style-guide.md), but we try to use the [Pythonic `lower_case_with_underscores` naming convention](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) for functions and variables.

Reply:
@dav3r Addressed in this [commit](https://github.com/cisagov/pe-reports/pull/106/commits/44b345ab38b499d181dbc95e3dfe43e001650cbb).

Reply:
That looks much better! 👍